src/routes/(console)/organization-[organization]/teamReadonlyAlert.svelte

```diff
@@ -0,0 +1,29 @@
+<script lang="ts">
+    import { page } from '$app/stores';
+    import { Button } from '$lib/elements/forms';
+    import { organization } from '$lib/stores/organization';
+    import { HeaderAlert } from '$lib/layout';
+    import {
+        billingLimitOutstandingInvoice,
+        hideBillingHeaderRoutes,
+        readOnly,
+        teamStatusReadonly
+    } from '$lib/stores/billing';
+    import { base } from '$app/paths';
+
+    $: redirectUrl = `${base}/organization-${$organization.$id}/billing#payment-history`;
+</script>
+
+{#if $organization?.$id && $organization?.status === teamStatusReadonly && $organization?.remarks === billingLimitOutstandingInvoice && $readOnly && !hideBillingHeaderRoutes.includes($page.url.pathname)}
+    <HeaderAlert type="error" title="Access restricted">
+        <svelte:fragment>
+            Access to your organization resources has been restricted as you have unpaid invoices.
+            Please pay the invoice to restore access.
```

Comment:
is this copy cleared by design?

Reply:
not yet, I'll have a setup for design to review.